index.bs

```diff
@@ -559,31 +583,60 @@ Width of the play-area bounds in meters. The bounds are defined as an axis-align
 <dfn attribute for="VRStageParameters">sizeZ</dfn>
 Depth of the play-area bounds in meters. The bounds are defined as an axis-aligned rectangle on the floor. The center of the rectangle is at (0,0,0) in standing-space coordinates. These bounds are defined for safety purposes. Content should not require the user to move beyond these bounds; however, it is possible for the user to ignore the bounds resulting in position values outside of this rectangle.
 
-
-## Navigator Interface extension ## {#interface-navigator}
+## VR Interface ## {#interface-vr}
 
 <pre class="idl">
-partial interface Navigator {
-  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getVRDisplays();
-  readonly attribute FrozenArray&lt;VRDisplay&gt; activeVRDisplays;
+interface VR : EventTarget {
+  Promise&lt;boolean&gt; getAvailability();
+  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getDisplays();
+
+  attribute EventHandler ondisplayconnect;
```

Comment:
@cvan , Are you suggesting we add `displayadd` in addition to `displayconnect` or replace `displayconnect` with `displayadd`?  In your mind, what is the difference between 'adding' a display and 'connecting' a display?